app/routes/docs/producers.mdx

```diff
@@ -0,0 +1,87 @@
+---
+handle:
+  breadcrumb: New Producers
```

Comment:
This should match the level 1 heading on the page and the link text in the side nav.